packages/broker/src/plugins/testnetMiner/TestnetMinerPlugin.ts

```diff
@@ -57,6 +58,10 @@ export class TestnetMinerPlugin extends Plugin<TestnetMinerPluginConfig> {
         if (this.pluginConfig.stunServerHost !== null) {
             this.natType = await this.getNatType()
         }
+        this.networkNode.setExtraMetadata({
+            natType: this.natType || null,
```

Comment:
`this.natType` may be `undefined`. The problem was that when this was the case, the serialized version of JSON did not contain the key `natType` at all!